dev/docker/cudf/Dockerfile

```diff
@@ -12,7 +10,8 @@ ENV PATH=$SPARK_HOME/bin:$PATH
 ENV CUDA_ARCHITECTURES=70
 
 WORKDIR /opt/gluten
-RUN ./dev/buildbundle-veloxbe.sh --run_setup_script=ON --build_arrow=ON --spark_version=3.4 --enable_gpu=ON && \
+RUN bash ./dev/buildbundle-veloxbe.sh --run_setup_script=OFF --build_arrow=ON --spark_version=3.4 --enable_gpu=ON && \
     touch cudf.sql && echo "select o_orderkey from orders order by o_orderkey;" > cudf.sql
 
 # You can try the data in folder backends-velox/src/test/resources/tpch-data-parquet
+RUN rm -rf /opt/gluten
```

Comment:
The cudf_build is time-consuming, do we remove the cache?

Reply:
this will remove the velox/gluten build, the folder after build is nearly 25G
`/root/.ccache` is kept.